scalableminds/webknossos
# Enable hdf5 mapping in isosurface computation

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- enable hdf5 mapping, request isosurface (shift + click after enabling in settings). Should give mapped isosurface
- isosurface with json mapping should also still work

### Issues:
- fixes #4509

------
